sdk/react/ooyalaSkinPanelRenderer.js

```diff
@@ -132,7 +132,8 @@ OoyalaSkinPanelRenderer.prototype.renderVideoView = function() {
         upNext: this.skin.props.upNext,
         icons: this.skin.props.icons,
         adScreen: this.skin.props.adScreen,
-        live: this.skin.props.live
+        live: this.skin.props.live,
+        seekFwd: this.skin.props.seekFwd
```

Comment:
Use skipcontrols instead if adding new config in skin.
https://github.com/ooyala/skin-config/blob/master/skin.json#L181